Tasks/NuGetInstaller/task.json

```diff
@@ -79,25 +79,30 @@
                 "Normal": "Normal",
                 "Detailed": "Detailed"
             }
+        },   
+        {
+            "name": "nuGetVersion",
+            "type": "radio",
+            "label": "NuGet Version",
+            "defaultValue": "3.3.0",
+            "helpMarkDown": "The version of NuGet to use, or external verson.",
+            "required": "true",
+            "groupName": "advanced",
+            "options": {
+                "3.3.0": "3.3.0",
+                "3.5.0.1829": "3.5.0 – build 1829 (rc2)",
```

Comment:
@bryanmacfarlane they can't type in other versions, these are the two exact ones sent down with the task.  Their other option is a full path to a custom nuget.exe they've already acquired so we couldn't do the full combo box solution without some confusion.  We do not want to be downloading nuget in-task, that should be something leaning on agent-wide binary acquisition and caching (and proxy management), or if those don't line up then some other common similar system.  To date, if people have needed to fill in that Custom box then they acquire nuget through a previous task or agent manipulation.  You make a good point about the value.  I'll spin with my team -- we'll either have to persist that as the value even with a new 3.5 and let it not match the actual version of 3.5 we ship beneath, or change this to 3.5 as the value now and not match the binary ever.  We can manage it either way but good callout.  
@chrisrpatterson This is a requirement from nuget.  Timelines did not line up for our RC2 release and this specific build is the *only* one that works onprem against an onprem packaging service due to required auth changes in nuget.exe.  Because this isn't the rtm, or even an official rc2 (it is not on dist), we are specifying a very specific string here.  We must be clear to those opting-in that this is prerelease nuget.  In the next 3 weeks we should have an actual RTM at which point we'll replace the ux label with 3.5 (or similar).